core/primitives/res/README.md

```diff
@@ -8,10 +8,16 @@ All parameter value to configure the runtime are defined in `parameters.txt`.
 Parameters added or changed in protocol upgrades are defined in differential
 config files with a naming scheme like `V.txt`, where `V` is the new version.
 
-The content of the initial config files and the diff is always just one flat
-list of typed keys and untyped values. Key names are defined in
+The content of the base configuration file is one flat list of typed keys and
+untyped values. Key names are defined in
 `core/primitives-core/src/parameter.rs`.
 
+The format of the differential files is slightly different. While the keys are
+the same, values that change use a format lie this: `key: old_value ->
+new_value`. Deleting a parameter value for the new version is done as follows:
+`key: old_value ->`. Finally, inserting is done using the same syntax as in the
```

Comment:
It isn't super clear to me what “deleting” means here. My intuition says that the merged configuration will be missing this parameter which will in turn use the default value for the parameter, which can potentially be quite different semantically from deleting entirely.

Reply:
Yes you are right, so I changed the wording to make it clearer.